src/Helpers/PConstants.js

```diff
@@ -167,6 +167,9 @@ module.exports = {
     SPHERE:         40,
     BOX:            41,
 
+    // Arc mode
+    CHORD:          2,
```

Comment:
We can add the OPEN and PIE constants here, too, even if OPEN does what we already do, and the PIE constant doesn't affect rendering (yet)